containers/server-attestation-image/server-attestation-image.changes

```diff
@@ -0,0 +1,9 @@
+Tue Feb 13 16:59:23 CET 2024 - kwalter@suse.com
```

Comment:
this is wrong folks.

the changelog must  follow a chronological  order  with  oldest entries on the bottom.

Reply:
ignore the last commit for now it's just temporary to see what is needed to make the test setup happy